front/src/applications/operationalStudies/views/Scenario.tsx

```diff
@@ -250,7 +250,7 @@ export default function Scenario() {
                 />
               </div>
             </div>
-            <div className={collapsedTimetable ? 'col-lg-12' : 'col-lg-8'}>
+            <div className={collapsedTimetable ? 'col-12' : 'col-hdp-9 col-xl-8 col-lg-7 col-md-6'}>
```

Comment:
can’t we just use a constant width in the range 768-1920 ? it’s okayish best, way too large at worst.

Reply:
The whole point of these new breakpoints is to tailor them to users' different screens, isn't it?